test/jdk/ProblemList.txt

```diff
@@ -470,9 +470,6 @@ sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java
 java/awt/Window/8159168/SetShapeTest.java 8274106 macosx-aarch64
 java/awt/image/multiresolution/MultiResolutionJOptionPaneIconTest.java 8274106 macosx-aarch64
 
-# This test fails on macOS 14
-java/awt/Choice/SelectNewItemTest/SelectNewItemTest.java 8324782 macosx-all
-
```

Comment:
This file should be identical to 9efecc96faa6d0b52a4555ee036c4dce6f4658a3.

Reply:
https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/9efecc96faa6d0b52a4555ee036c4dce6f4658a3 was following a few changes https://github.com/openjdk/jdk23u/commits/master/test/jdk/ProblemList.txt

GIT missed the latest changes https://github.com/ibmruntimes/openj9-openjdk-jdk23/pull/58#discussion_r1803815531 when merging the current OpenJDK contents to `master` branch.
The change is made manually in this PR to keep `openj9-staging` consistent with OpenJDK updates.

Reply:
Those subsequent changes are from commits that are not ancestors of [jdk-23.0.1+11](https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/df073d9f7e95611bf43d61c70add424b9bc3b9a7) and so don't belong in this pull request which claims to merge up to that tag.

Reply:
Restored [jdk-23.0.1+11](https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/df073d9f7e95611bf43d61c70add424b9bc3b9a7) content.